cmd/autoowners/main.go

```diff
@@ -390,7 +406,11 @@ func main() {
 		logrus.WithError(err).Fatal("Failed to change to root dir")
 	}
 
-	if err := pullOwners(gc, o.targetDir, sets.NewString(o.blacklist.Strings()...)); err != nil {
+	var configSubDirectories = o.configSubDirs.Strings()
+	if len(o.configSubDirs.Strings()) == 0 {
```

Comment:
nit: `if len(configSubDirectories)` is the same thing and IMO shows the flow better.